pkg/multiTenancyPlugins/authorization/defaultAuthZImpl.go

```diff
@@ -60,13 +61,15 @@ func (defaultauthZ *DefaultAuthZImpl) Handle(command utils.CommandEnum, cluster
 		//In case of container json - should record and clean - consider seperating..
 	case utils.CONTAINER_START, utils.CONTAINER_STOP, utils.CONTAINER_RESTART, utils.CONTAINER_DELETE, utils.CONTAINER_WAIT, utils.CONTAINER_ARCHIVE, utils.CONTAINER_KILL, utils.CONTAINER_PAUSE, utils.CONTAINER_UNPAUSE, utils.CONTAINER_UPDATE, utils.CONTAINER_COPY, utils.CONTAINER_CHANGES, utils.CONTAINER_ATTACH, utils.CONTAINER_LOGS, utils.CONTAINER_TOP, utils.CONTAINER_STATS, utils.CONTAINER_EXEC:
 		if !utils.IsResourceOwner(cluster, r.Header.Get(headers.AuthZTenantIdHeaderName), mux.Vars(r)["name"], "container") {
-			return errors.New("Not Authorized or no such resource!")
+			http.Error(w, fmt.Sprintf("No such container "), http.StatusNotFound)
+			return errors.New(fmt.Sprint("status ",http.StatusNotFound," HTTP error: No such container"))
 		}
```

Comment:
But in the executer you also have:
			http.Error(w, err.Error(), http.StatusBadRequest)
So what would you have on the response ?